### ISSUE TITLE
Packages to consider adding

### ISSUE THREAD

**Issue by ljlamarche** [EarthCubeInGeo/resen-core#42]
- [ ] sunpy
- [x] plasmapy
- [x] pydarn
- [x] vires client
- [x] `%matplotlib widget` for JupyterLab
- [x] visuamisr
- [x] hdf5 viewer?
- [ ] seaborn

**Comment by ljlamarche** [comment]
- [ ] pydarn

**Comment by asreimer** [comment]
@pmreyes2 tested vires and it worked fine right?

@ljlamarche and I have tested pydarn. Installation is easy and a non-issue.

**Comment by asreimer** [comment]
pip install git+https://github.com/asreimer/visuamisr.git@v2.0.3

**Comment by ljlamarche** [comment]
Some notes:
- plasmapy, pydarn, and viresclient all require python 3.6
- The latest version of sunpy is only available for python 3.6 and later.  We'll need to install version 0.9.10 in the py27 environment.
- I get a `UnicodeDecodeError` when trying to import sunpy.  @astib Did someone specifically request sunpy?  If not, I might recommend just opening an issue on the sunpy github page and saving it for a later release unless the problem is patched quickly.

Packages sucessfully installed in py36:
- plasmapy
- pydarn
- viresclient
- visuamisr

Packages successfully installed in py27:
- visuamisr

**Comment by astib** [comment]
I don't remember anyone specifically requesting sunpy. I like the current list of packages. Let's go ahead with these 4 for now, and leave sunpy for later.

**Comment by ljlamarche** [comment]
Great, thanks!  Also, do we want to try to update the packages in the py27 environment?  Our last release right before AGU would have been right before the python 2 official depreciation, so any package updates since then may not work in python 2.

**Comment by valentic** [comment]
This is one of those topics that we would want to have a 
migration/deprecation strategy. Even though we really don't have a lot 
of users, we should act like we do. Part of the release notes for this 
release should clarify that this will be the last one to support py27 
and that it will be removed in the next release (2020.agu) or something 
like that. For some of us (yes, me), I have a ton of py27 code.

Todd

**Comment by ljlamarche** [comment]
Following on to this, do we want to add additional environments?  They're up to python 3.8 now right, so do we want to add a py37 and py38 environment when we remove the py27 environment?

**Comment by ljlamarche** [comment]
I'm adding [seaborn](https://seaborn.pydata.org/) to the list of things for us to consider in the future.  I believe it's statistical data visualization similar to matplotlib.  It's actually a package that was imported in the original ICON tutorial, but I found the rest of the tutorial ran just fine without it, so I just removed the import statement.  Even so, we should look at if this is popular or commonly used in some parts of the community.
